cube/cube.frag.inc

```diff
@@ -1,41 +1,84 @@
-	// 7.9.2888
-	0x07230203,0x00010000,0x00080007,0x00000030,0x00000000,0x00020011,0x00000001,0x0006000b,
+	// 1115.1.0
```

Comment:
Wow that's a lot of versions of glslang

Reply:
I am unsure when the checked in code was compiled, because Vulkan-Tools used to compile glslang, then it went to just downloading the binary, to the current checked in state.